11ty/_includes/layouts/base.njk

```diff
@@ -1,17 +1,20 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://use.typekit.net/" crossorigin />
+    <link rel="dns-prefetch" href="https://use.typekit.net/" />
```

Comment:
This may seem redundant, but there's a reason for each of these hints:

1. `preconnect` will establish a complete connection to the origin specified in `href`.
2. Not all browsers support `preconnect`, but a larger number of browsers support `dns-prefetch`, which performs the DNS lookup rather than establish a connection to the remote server. Not as effective, but still gives something of a boost to older browsers.